### PR TITLE
Xiaozhua

### PR DESCRIPTION
**Pull request by youqingxiaozhua**
- 明确各状态输出的错误信息
- status <= 3的情况下都允许重新发送邮件
- 解决strftime不支持中文问题

### CODE REVIEW

File: pic_email.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 def write_to_pic(name, email, token):  # 执行完这个方法后生成一个 result.png 图片 可加入email参数
     number = get_number(email)  # 用这个方法获取到编号
-    date_str = datetime.datetime.strftime(datetime.datetime.now(),'%Y年%m月%d日') #获取日期
+    date_str = datetime.datetime.strftime(datetime.datetime.now(), u'%Y{0}%m{1}%d{2}').format(*'年月日')  # 获取日期
```
Comment (zhaofeng-shu33) [inline pic_email.py]: Do not use u'xxx'. This is Python 2 usage.

File: app.py [hunk 2]
```diff
@@ -80,36 +83,35 @@ def token():
...
-        return response  # Key被用过了
+    elif request.method == 'POST':
+        message = json.loads(request.get_data(as_text = True))
+        name = message.get('name')
+        token = message.get('token', '')
+        if name is None:
+            return response_json(code=1, message='Name is required')
+        result = confirm_token(token)
+        if result is False:
+            return response_json(code=2, message='Token is not available, plz check your latest email')
```
[the remark this answers is not in the document]
Comment (youqingxiaozhua) [reply app.py]: generally, any code except 0 will be considered as an error, and the code is used to distinguish between different errors. The best practice is using different codes for different errors.

File: pic_email.py (hunk 1 above)
Comment (youqingxiaozhua) [reply pic_email.py]: Oh, yes! I forget to delete it, it's useless in Python3.

File: app.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (zhaofeng-shu33) [reply app.py]: Ok, please update README.md accordingly.

**Pull request merged**